dask/widgets/templates/highlevelgraph_layer.html.j2

```diff
@@ -25,6 +25,19 @@
                         <td style="text-align: left;">{{ val }}</td>
                     </tr>
                 {% endfor %}
+                {% for dep in dependencies %}
+                    {% if loop.index > 1 %}
+                        <tr>
+                            <th></th>
+                            <td style="text-align: left;">{{ dep }}</td>
+                        </tr>
+                    {% else %}
+                        <tr>
+                            <th style="text-align: left; width: 150px;"> dependent on </th>
```

Comment:
```suggestion
                            <th style="text-align: left; width: 150px;"> depends on </th>
```
We use the term `dependency` to refer to the inputs of a layer; `dependent` refers to the outputs.